virttest/bootstrap.py

```diff
@@ -262,6 +262,40 @@ def get_directory_structure(rootdir, guest_file):
         previous_indent = indent
 
 
+def sync_download_dir(interactive):
+    base_download_dir = data_dir.get_base_download_dir()
+    download_dir = data_dir.get_download_dir()
+    logging.debug("Copying downloadable assets file definitions from %s "
+                  "into %s", base_download_dir, download_dir)
+    download_file_list = glob.glob(os.path.join(base_download_dir,
+                                                "*.ini"))
+    logging.debug("Donwload file list: %s", download_file_list)
+    for src_file in download_file_list:
+        dst_file = os.path.join(download_dir,
+                                os.path.basename(src_file))
+        if not os.path.isfile(dst_file):
+            shutil.copyfile(src_file, dst_file)
```

Comment:
create_config_file was more verbose here, but I don't see the urge to log this...